src/cpu.rs

```diff
@@ -285,13 +285,21 @@ impl Cpu {
 		self.mmu.tick(&mut self.csr[CSR_MIP_ADDRESS as usize]);
 		self.handle_interrupt(self.pc);
 		self.clock = self.clock.wrapping_add(1);
-		self.write_csr_raw(CSR_CYCLE_ADDRESS, self.clock);
+
+		// cpu core clock : mtime clock in clint = 8 : 1 is
+		// just an arbiraty ratio.
+		// @TODO: Implement more properly
+		self.write_csr_raw(CSR_CYCLE_ADDRESS, self.clock * 8);
```

Comment:
Oops, I have unexpectedly added this change in this commit although I was going to add in another commit (PR).